### DISCUSSION TITLE
✅ Make the Portfolio and Posts widgets have the same settings

### DISCUSSION THREAD

**Discussion by tilnox** [elementor/elementor#15480] [Feature Request]
Hello
I've found  Elementor due to a portfolio template i've aquired. Beautiful, so easy to use and edit, with all the things i needed as a designer+photographer, to build my portfolio. The possibilities were infinite, and i decided to go Pro and do my own thing.

With Elementor + Elementor Pro, startinf from scratch, to my surprised, the features that made it easy to build a portolio, were not available anymore. What happend was, the plugin was edited by the theme creator with features that he liked, making it unique.

I tryed to recreate what i needed, with the **Elementor** i've just bought, but was not possible to make. Asked for help, maybe i just couldn't find it or know how.. but the answer was "that's not available".

So here are the request features:

### **Meta Outside/Inside**

I try to build it with the pro version, and its tricky. Users can create a porfolio thumbnail listing with "portfolio" or "post" but...

![meta outside](https://user-images.githubusercontent.com/49846018/57328029-3b4ba100-7108-11e9-8d3c-24f0adcf47d6.png)

**with portfolio** - _content source "porfolio"(elemento pro):_

- **tilte** is available but not **category**
- **title** location option, only inside
- **alignment**: only center/center

**with posts** - _content source "porfolio" (elemento pro):_

- **tilte** is available but not **category**
-**title location** -only outside
-**adding metada** is available, but only for blog type content (_author, date, time, comments_, again not category)

**Is Category available anywhere on Elementor?**

Yes! if you decide to show "filter bar", "taxonomy" has the options i'm talking about: "_categories, tags, product categories, product tags, porfolio category_" they are just missing on other places, like in dynamic content.

Bulding a detal page with dynamic content:

-  **title** (available)
-  **category** (is not listed. actually everthing available as an option on taxonomy dropdown, is not available as dynamic content, and it could be)

### **Requests:**

- Add **category** to show/hide on portfolio thumbnail
![meta on off](https://user-images.githubusercontent.com/49846018/57348629-b768d780-714e-11e9-8966-ab40e09dd5e4.png)

- Add **metadata** inside/outside as an option
![inside outside](https://user-images.githubusercontent.com/49846018/57348608-a324da80-714e-11e9-92af-4c1bdcccd09a.png)

- Allow **taxonomy** and filter query to be available as dynamic content and metadata
![Screen Shot 2019-05-08 at 05 05 55](https://user-images.githubusercontent.com/49846018/57348735-129aca00-714f-11e9-84d0-59e2cb33456b.png)
![image](https://user-images.githubusercontent.com/49846018/57348857-bb492980-714f-11e9-9eda-2e2d69a1110e.png)

-----------------------------------------------------------------

### **Layout type**

If i want do build something like this with Elementor pro, is not possible.  Here's how it could:
![content with column size](https://user-images.githubusercontent.com/49846018/57334916-0136cb00-7119-11e9-881d-042b9403f869.png)

Adding a new layout type beside masonory. In this version its call "metro", and it lets you choose the "posts" you want to share, one my one, and lets you change the amount of colums  each post occupies. 
![columns](https://user-images.githubusercontent.com/49846018/57335955-2e38ad00-711c-11e9-943a-a027ef2eb303.png)

**Why is this option a _must have_?**

- lets users choose which projects have focus, playing with size.
- it lets users edit that _out of the box_ look, great for designers and photographers
- and off course its a trend, and Elementor should be helping users to easyly recreate it :)

-----------------------------------------------------------------

### **Meta On/Off**

Doing  something like this with **Elementor Pro**: portfolio projects or posts (titles), with large bold and beautiful full screen font. 
![new_desktop_2x](https://user-images.githubusercontent.com/49846018/57341391-2d127a80-7132-11e9-825e-c7c4efefe193.jpg)
_exemple from https://dribbble.com/shots/5585752-Alchemy-Project_

**Is it possible?**
In teory.. kind of.  **Elementor**  has all the elements and tools to do it, but it reality you can't..
Because there's metadata you can't turn on/off. With the Elementor version i have, its possible. Here's how:
![metadata on off](https://user-images.githubusercontent.com/49846018/57342075-01dd5a80-7135-11e9-8e42-85029247e57f.png)

Because the focus was on developing the **"portfolio" Element**: _thumbnail, tiltle and categories_ looks like the obvious choices, but Elementor can go for **dynamic content** too. If you're using **Elementor "Posts"**, there's a lot of meta you can turn on/off, like: _date, excerpt, read more, comments_... But if you using **"porfolio"** there's not a lot you can edit , without looking like every portfolio page.

**Why is this an important feature to add?**
It's minimal, elegant, font oriented,  and a strong UI solutions. It would be great to create them effortless with Elementor. 

![squarespace](https://user-images.githubusercontent.com/49846018/57343455-5f74a580-713b-11e9-9dc5-40b386e64715.png)
exemple from https://foundry-demo.squarespace.com/

-----------------------------------------------------------------

Sorry for the long post, the requests are all related to the _**porfolio element**_. I alreay have my hands on a version with most of the features requested, so i know how fantastic and easy it would be to have them on the free or the Pro Elementor version. 

Thank you

**Comment by sommarzan** [comment]
+1

**Comment by studio-m2v** [comment]
+1

**Comment by Jejito** [comment]
+1 
Options for customizing the portfolio widget are way too limited. At the very least we should be able to show/hide category, description, tags along the title, and have some more options regarding alignment and position.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro versions, 3.8.0 (Loop Builder), and 3.15.0 (Taxonomy Filter)! 🥳 

You can now build your own Loop Design with as many dynamic tags and information as you want. You can also now Filter the Loop Grid with the new experimental Taxonomy Filter widget.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
